npm/ng-packs/packages/schematics/src/commands/change-theme/style-map.ts

```diff
@@ -0,0 +1,223 @@
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/side-menu/layout-bundle.css',
+    inject: false,
+    bundleName: 'layout-bundle',
+  },
+  {
+    input: 'node_modules/@volosoft/abp.ng.theme.lepton-x/assets/css/abp-bundle.css',
+    inject: false,
+    bundleName: 'abp-bundle',
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/dark.rtl.css',
+    inject: false,
+    bundleName: 'dark.rtl',
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/light.rtl.css',
+    inject: false,
+    bundleName: 'light.rtl',
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/dim.rtl.css',
+    inject: false,
+    bundleName: 'dim.rtl',
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/bootstrap-dim.rtl.css',
+    inject: false,
+    bundleName: 'bootstrap-dim.rtl',
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/bootstrap-dark.rtl.css',
+    inject: false,
+    bundleName: 'bootstrap-dark.rtl',
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/bootstrap-light.rtl.css',
+    inject: false,
+    bundleName: 'bootstrap-light.rtl',
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/ng-bundle.rtl.css',
+    inject: false,
+    bundleName: 'ng-bundle.rtl',
+  },
+  {
+    input: 'node_modules/@volosoft/ngx-lepton-x/assets/css/side-menu/layout-bundle.rtl.css',
+    inject: false,
+    bundleName: 'layout-bundle.rtl',
+  },
+  {
+    input: 'node_modules/@volosoft/abp.ng.theme.lepton-x/assets/css/abp-bundle.rtl.css',
+    inject: false,
+    bundleName: 'abp-bundle.rtl',
+  },
+]);
+styleMap.set(ThemeOptionsEnum.LeptonXLite, [
+  {
+    input: 'node_modules/@volo/ngx-lepton-x.lite/assets/css/bootstrap-dim.css',
+    inject: false,
+    bundleName: 'bootstrap-dim',
+  },
+  {
+    input: 'node_modules/@volo/ngx-lepton-x.lite/assets/css/ng-bundle.css',
+    inject: false,
+    bundleName: 'ng-bundle',
+  },
+  {
+    input: 'node_modules/@volo/ngx-lepton-x.lite/assets/css/side-menu/layout-bundle.css',
+    inject: false,
+    bundleName: 'layout-bundle',
+  },
+  {
+    input: 'node_modules/@abp/ng.theme.lepton-x/assets/css/abp-bundle.css',
+    inject: false,
+    bundleName: 'abp-bundle',
+  },
+  {
+    input: 'node_modules/@volo/ngx-lepton-x.lite/assets/css/bootstrap-dim.rtl.css',
+    inject: false,
+    bundleName: 'bootstrap-dim.rtl',
+  },
+  {
+    input: 'node_modules/@volo/ngx-lepton-x.lite/assets/css/ng-bundle.rtl.css',
+    inject: false,
+    bundleName: 'ng-bundle.rtl',
+  },
+  {
+    input: 'node_modules/@volo/ngx-lepton-x.lite/assets/css/side-menu/layout-bundle.rtl.css',
+    inject: false,
+    bundleName: 'layout-bundle.rtl',
+  },
+  {
+    input: 'node_modules/@abp/ng.theme.lepton-x/assets/css/abp-bundle.rtl.css',
+    inject: false,
+    bundleName: 'abp-bundle.rtl',
+  },
+]);
+// the code written by Github co-pilot. thank go-pilot. You are the best sidekick.
```

Comment:
i think it should co-pilot instead of go-pilot

Reply:
Nice catch but no, brand is co-pilot
![image](https://user-images.githubusercontent.com/2217899/198036510-c2ee7be2-f1c4-49b5-b253-6ed569a768aa.png)